tools/submit_job.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+
+import boto3
+
+from time import sleep
+
+BATCH_SUCCESSFUL_STATUS = 'SUCCEEDED'
+BATCH_FAILURE_STATUS = 'FAILED'
+BATCH_PROCESSED_STATUSES = [BATCH_FAILURE_STATUS, BATCH_SUCCESSFUL_STATUS]
+
+batch_client = boto3.client('batch', region_name=os.getenv('AWS_REGION', 'eu-west-2'))
+
+
+def print_status(current_status, count):
+    progress = '.' * count
+
+    sys.stdout.write('Batch status: %s\n' % current_status.ljust(10))
```

Comment:
Since this looks similar to a block from the e2e tests from the reporter, is it worth also adding / including the comment added there to explain what this is doing to those of us less fluent in shell scripting?